mainApp/src/main/java/logisticsmarshall/tqs/ua/model/Reputation.java

```diff
@@ -30,7 +30,7 @@
     private Delivery delivery;
 
     @ManyToOne
-    @JoinColumn(name = "driver_id", nullable = false)
+    @JoinColumn(name = "driver_id", nullable = true)//maybe we take really long to assign a driver
```

Comment:
A reviews não deviam ser só depois do driver acabar a entrega? Logo isto nunca irá acontecer

Reply:
é possivel um user dar review se nunca for atendido, e isto facilita imenso o sistema, sem isto tenho de meter um monte de logica extra